src/services/expenses.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+let expenses = [];
+
+const reset = () => {
+  expenses = [];
+};
+
+const getAll = () => expenses;
+
+const getAllFilteredExpenses
+  = ({ userId, categories, from, to }) => {
+    if (!expenses) {
+      return [];
+    }
```

Comment:
is it correct check? because `expenses` all time `true`, even if it is []

Reply:
agree